### ISSUE TITLE
When I set my minecraft window height to 985, 997, 998, 1002, 1004, or 1005 pixels (but no other values), the sides of the crosshair+ disappear

### ISSUE THREAD

**Issue by leijurv** [ImpactDevelopment/ImpactClient#727]
<img width="91" alt="screen shot 2018-10-02 at 11 05 06 am" src="https://user-images.githubusercontent.com/3837873/46378904-f7645b00-c651-11e8-908b-08c57736a29e.png">
I checked all possible heights, the crosshair renders properly at height 986 through 996, 999, 1000, 1001, and 1003, and all heights 984 and below. My width is 1680.

**Comment by leijurv** [comment]
Never mind actually as I make it smaller it flickers a ton, so there's a lot of heights that don't work in the 400 to 600 range.

**Comment by VitorISs** [comment]
so... shouldn't this be closed ?

**Comment by leijurv** [comment]
No, there are still many heights that cause the sides of the cross hair to disappear, that's the issue here. @VitorISs

**Comment by VitorISs** [comment]
HmMmMMmm now i got it

**Comment by ZeroMemes** [comment]
Does this still happen in 4.4?

**Comment by leijurv** [comment]
Yes, with OptiFine on and off, and with antialiasing on and off. Only happens when width < 2. However it should just draw a 1 pixel line and not randomly disappear.

However, the heights at which it disappears have changed. For example, now it disappears at 1680x996 but not 1680x997 whereas in 4.3 it was the other way around.

**Comment by leijurv** [comment]
<img width="841" alt="screen shot 2018-10-16 at 7 48 20 pm" src="https://user-images.githubusercontent.com/3837873/47059055-7c845f80-d17c-11e8-9b56-0d42cd3e0f9a.png">
<img width="850" alt="screen shot 2018-10-16 at 7 48 36 pm" src="https://user-images.githubusercontent.com/3837873/47059057-7e4e2300-d17c-11e8-98db-af1c85b241e1.png">

Here you can see the sides of the crosshair present at 1680x997 but missing at 1680x996.

**Comment by LeafHacker** [comment]
Could be a GUI scale rounding issue?

**Comment by leijurv** [comment]
No the crosshair isn't affected by gui scale. It's just garbage 😠
